standard_releases.md

```diff
@@ -97,32 +97,12 @@ least work! The general process is:
 * Review, test and merge those changes into the `master` branch.
 * Call a "freeze" on the `master` branch. This freeze lasts during the translation
   process and applies to the schema and docs folders in the repository.
-* Begin translation of the new release (see [translations](/translations)) using the
+    * See below for how to make necessary edits during a "freeze".
+* Carry out translation of the new release (see [translations](/translations)) using the
   BODS-main project in Transifex.
-* During translation there may be a need to fix issues that exist on the `master`
-  branch. If they are outside schema and docs they can be developed on a branch
-  taken from `master` and merged back in when completed.
-  If the changes affect schema and/or docs then they may, in turn, affect the
-  translation. The process for making these changes is:
-  * First create an issue in Github to track the problem.
-  * Then create a new branch that uses the issue number, e.g.
-    `297-concept-translation`.
-  * When the branch is ready to be merged into `master`, ensure that the translations
-    on the new branch are either equivalent to or ahead of the translations on
-    `master`. ie. if no new translations have been merged back into `master` since the
-    new branch was made, no need to pull new translations down from Transifex. But if
-    `master` has since been updated, be sure to fetch the new translations to the new
-    branch, either from Transifex or by rebasing against `master` (the best option will
-    depend on the changes you have made in the new branch).
-  * Merge the branch (`297-concept-translation`) into `master`.
-  * Checkout the `master` branch, extract the new or changed strings per the translation
-    process, and push them up to Transifex.
-  * Wait for the new changes to be translated.
-  * Once the translation is finalised, check out a new branch for the translations and
-    pull them from Transifex. Commit these and make a PR against `master` to bring it up
-    to date.
-* Once the release is finalised and everything is ready, create a branch for the
-  new version (eg, `0.2.0`) from `master`.
+* Once the release is finalised, and the translations have been pulled from
```

Comment:
I think we are listing things out of order here. How about something like:

> * Once the translation is finished, pull the translated strings down from Transifex and merge them into master.
> * Create a branch for the new version (e.g. `0.2.0`) from `master`.
> * The release is now finalised

(Although I'm not sure how useful it is for us to be referring to the release being finalised without explaining what we mean.)